source/data-model-partials/data-model.md

```diff
@@ -0,0 +1,99 @@
+## Data model diagram
+This diagram was correct as of 27th February 2018, and is already slightly out of date. It should be useful to see the 
```

Comment:
I would drop the `already slightly out of date` bit here. Just a "correct as of" is fine.

I think we also probably want to set a page expiry on this section, as an ongoing reminder that we need to keep this updated. See https://tdt-documentation.london.cloudapps.digital/page-expiry.html#page-expiry-and-review for how.

Reply:
I think we need to add `show_expiry: true` to the global config for this to work.

Reply:
It works for me locally - it's at the bottom of the page.  I think you have o add stuff to *not* show the banner.

Reply:
Oh it's not in the review app though. Weird.

Reply:
The typos are still there so maybe it's just being slow to release 🤔

Reply:
Maybe... but it looked like it was out alredy in Heroku pipeline view.  I've pushed a version with the config setting anyway, probably doesn't hurt.